### ISSUE TITLE
När man skriver beskrivning och klickar på enter så följer inte scrollen med

### ISSUE THREAD

**Issue by ThomasAxakon** [simpell-dev/simpell-www-bugs#79]
Skapa annons
Förbättring
Chrome
När man skriver sin beskrivning och göra massa enter slag så följer inte scrollen med förräns man skriver något igen. Scrollen borde alltid följa med där markören är.

**Comment by PeterLAxakon** [comment]
Det gör den hos mig. Kan du bifoga en bild eller testa igen(sitter med chrome)

**Comment by ThomasAxakon** [comment]
[Screen Recording 2020-09-04 at 13.04.33.zip](https://github.com/simpell-dev/simpell-www-bugs/files/5174393/Screen.Recording.2020-09-04.at.13.04.33.zip)
Här är en screenrecording på hur det går för mig, den blinkande indikatorn vart man skriver "försvinner" nedanför textfältet

**Comment by PeterLAxakon** [comment]
wow, jag får inte samma beteende här. Sitter du på en mack, detta får vi kika på senare

**Comment by ThomasAxakon** [comment]
Sitter på en Mac @PeterLAxakon, vi kanske får inkorporera vilken dator man är på med i feedback dokumentet?

**Comment by PeterLAxakon** [comment]
Ja, bra ide, utöka formuläret med OS/dator, detta är en jäkligt skum grej, kan du testa i firefox ?
